tracing-attributes/tests/ui.rs

```diff
@@ -0,0 +1,7 @@
+// Only test on nightly, since UI tests are bound to change over time
```

Comment:
hmm, nightly builds are currently allowed to fail on our CI setup, to allow continuing to merge PRs in the face of nightly regressions...will this still catch changes that break the error output, given that?

Reply:
probably not. I could pin this to beta or stable instead? the important part it needs to be restricted to _some_ version because the diagnostics change between releases quite often.

Reply:
Done